maintainers/maintainer-list.nix

```diff
@@ -8879,6 +8879,12 @@
     githubId = 649832;
     name = "Teo Klestrup Röijezon";
   };
+  terin = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add terin``.